src/log.cpp

```diff
@@ -310,16 +313,29 @@ void Logger::logToOutputs(LogLevel lev, const std::string &combined,
 //// *LogOutput methods
 ////
 
-void FileLogOutput::open(const std::string &filename)
+void FileLogOutput::setFile(const std::string &filename)
 {
-	m_stream.open(filename.c_str(), std::ios::app | std::ios::ate);
+	actionstream << "Log messages are saved to " << filename << std::endl;
+
+	std::ifstream ifile(filename.c_str(), std::ios::binary | std::ios::ate);
+	bool truncate = ifile.tellg() > DEBUGFILE_SIZE_MAX;
```

Comment:
`truncate` is a bit misleading. How about `is_too_large`?

Reply:
fixed